ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/SignatureIntegrationForm.tsx

```diff
@@ -257,6 +292,218 @@ function SignatureIntegrationForm({
+                                                                        />
+                                                                    </FormLabelGroup>
+                                                                </FlexItem>
+                                                            </Flex>
+                                                            {isEditable && (
+                                                                <FlexItem>
+                                                                    <Button
+                                                                        variant="plain"
+                                                                        aria-label="Delete header key/value pair"
+                                                                        style={{
+                                                                            transform:
+                                                                                'translate(0, 42px)',
+                                                                        }}
+                                                                        onClick={() =>
+                                                                            arrayHelpers.remove(
+                                                                                index
+                                                                            )
+                                                                        }
+                                                                    >
+                                                                        <TrashIcon />
+                                                                    </Button>
+                                                                </FlexItem>
+                                                            )}
+                                                        </Flex>
+                                                    </Flex>
+                                                )
+                                            )}
+                                        {isEditable && (
+                                            <Flex>
+                                                <Button
+                                                    variant="link"
+                                                    isInline
+                                                    onClick={() =>
+                                                        arrayHelpers.push({
+                                                            certificateChainPemEnc: '',
+                                                            certificatePemEnc: '',
+                                                            certificateOidcIssuer: '',
+                                                            certificateIdentity: '',
+                                                        })
```

Comment:
To help TypeScript help us, what do you think about something like

```ts
const defaultValuesOfCosignCertificateVerification: CosignCertificateVerification = {
    certificateChainPemEnc: '',
    certificatePemEnc: '',
    certificateOidcIssuer: '',
    certificateIdentity: '',
};
```

following assignment to `defaultValues`